toolkit/build.xml

```diff
@@ -28,6 +28,15 @@
    <mkdir dir="${tk}/license"/>
    <copy file="${streamsx.topology}/LICENSE" todir="${tk}/license"/>
 
+   <copy todir="${tk}/com.ibm.streamsx.topology.functional.python2">
+     <fileset dir="${tk}/com.ibm.streamsx.topology.functional.python" 
```

Comment:
Why three filesets, won't this result in copying everything?

Doesn't excludes support multiple items so a single fileset can be used?